PurchasesCoreSwift/Purchasing/ProductInfo.swift

```diff
@@ -30,7 +30,7 @@ import StoreKit
         case freeTrial = 2
     }
 
-    @available(iOS 11.2, macOS 10.13.2, *)
+    @available(iOS 11.2, macOS 10.13.2, tvOS 11.2, *)
```

Comment:
should this one also have the watch?

Reply:
I forgot to add it back when I was debugging my issues, thank you!